### ISSUE TITLE
Any ideas how to handle permissions? See details

### ISSUE THREAD

**Issue by amankkg** [acacode/swagger-typescript-api#239]
We have a permission system in our backend done via enum per each endpoint. I'd like to somehow automate such logic.

One way of doing so if swagger will have enum string as a note to an endpoint like this

![image](https://user-images.githubusercontent.com/3933028/114813143-466ad980-9dd3-11eb-97bb-11668e96966b.png)

Is there a way to add that endpoint as TS literal type to a resulting class or method?
Or, maybe there are other ways to achieve a similar outcome?

**Comment by js2me** [comment]
You can do any kind of manipulations with custom templates using `--templates` option

**Comment by amankkg** [comment]
Yeah, thanks, I've been looking through it. If you don't mind keeping this issue open then I'd like to reply here once I'll get something viable

**Comment by js2me** [comment]
@amankkg 
Can you show result code which you want to have ?   
Also I need part of your swagger schema
